ide/editor.settings.storage/src/org/netbeans/modules/editor/settings/storage/fontscolors/ColoringStorage.java

```diff
@@ -289,9 +293,11 @@ public void run() throws IOException {
     private static final String HIGHLIGHTING_FILE_NAME = "editorColoring.xml"; // NOI18N
     
     private static final String E_ROOT = "fontscolors"; //NOI18N
+    private static final String E_COLORREF = "colorref"; //NOI18N
```

Comment:
It seems that this is the element that _defines_ the color reference, rather than uses it. I would call that "colordef" rather than "colorref", or something similar. The "reference" is the place that refers to the previously defined color.

Reply:
Changed.